socialhome/streams/app/components/FollowedStampedElement.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <generic-stamped-element :title="title" :help-text="helpText" />
+</template>
+
+<script>
+import Vue from "vue"
+import GenericStampedElement from "streams/app/components/GenericStampedElement.vue"
```

Comment:
`Generic` kind of keeps colliding with me every time I look at this. Can we use `BaseStampedElement`? "Base" is a more common word to use in these cases for example with Django templates or classes.